uploader.json

```diff
@@ -2095,6 +2095,11 @@
             "pattern": [],
             "symbolic_name": "scsi"
         },
+		{
+            "file": "/proc/net/sctp/assocs",
+            "pattern": [],
+            "symbolic_name": "sctp_asc"
+		},
```

Comment:
@vishwanathjadhav please fix your indentation, it looks like you may be using tabs instead of spaces.

Reply:
Done